packages/cli/src/commands/upgrade/upgrade.js

```diff
@@ -46,7 +45,14 @@ const getPatch = async (currentVersion, newVersion, config) => {
   logger.info(`Fetching diff between v${currentVersion} and v${newVersion}...`);
 
   try {
-    patch = await fetch(`${rawDiffUrl}/${currentVersion}..${newVersion}.diff`);
+    const {status, data} = await fetch(
+      `${rawDiffUrl}/${currentVersion}..${newVersion}.diff`,
+    );
+
+    if (status < 200 || status > 299) {
+      throw new Error(`Failed to load page, status code: ${status}`);
```

Comment:
Users of `upgrade` command have no idea what page failed to load, status code doesn't tell much as well. Can you make this error a bit more descriptive, and maybe show the error message as well? Also, use `CLIError`.